torch/csrc/distributed/c10d/comm.cpp

```diff
@@ -20,7 +21,7 @@ class BroadcastWork {
         flat_tensor_({torch::utils::flatten_dense_tensors(bucket_tensors_)}) {
     BroadcastOptions broadcastOptions;
     broadcastOptions.rootRank = root_rank;
-    work_ = process_group->broadcast(flat_tensor_, broadcastOptions);
+    work_ = ops::broadcast(process_group, flat_tensor_, broadcastOptions);
```

Comment:
I think for DDP, we should just register a `_broadcast_coalesced` op and directly call that from python here: https://github.com/pytorch/pytorch/blob/master/torch/distributed/utils.py#L131. Can be done in a separate PR.

Reply:
I feel it is cleaner for python to just call into an aten op. In this case, DDP is calling _broadcast_coalesced and there is a bunch of python and C++ logic before we actually call into the `broadcast` cc op.